src/System.Net.Security/src/System/Net/Security/SslApplicationProtocol.cs

```diff
@@ -2,139 +2,111 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
 using System.Text;
 
 namespace System.Net.Security
 {
     public readonly struct SslApplicationProtocol : IEquatable<SslApplicationProtocol>
     {
-        private readonly ReadOnlyMemory<byte> _readOnlyProtocol;
         private static readonly Encoding s_utf8 = Encoding.GetEncoding(Encoding.UTF8.CodePage, EncoderFallback.ExceptionFallback, DecoderFallback.ExceptionFallback);
+        private static readonly byte[] s_http2Utf8 = new byte[] { 0x68, 0x32 };
```

Comment:
would it make sense to use Encoding.ASCII.GetBytes("h2") and similar below for clarity?

Reply:
Or just add a comment at the end of this line:

```c#
// Encoding.ASCII.GetBytes("h2")
```

Reply:
Ok